src/npc.cpp

```diff
@@ -1172,7 +1172,7 @@ void npc::make_angry()
         return; // We're already angry!
     }
 
-    add_msg( m_debug, "%s gets angry", name.c_str() );
+    add_msg( "%s gets angry", name.c_str() );
```

Comment:
This message was in debug because it has no sight handling and isn't untranslated (lacks the `_( )` around it).